WebApp/package.json

```diff
@@ -59,15 +59,13 @@
     "@material-ui/core": "^4.0.1",
     "@material-ui/icons": "4.0.1",
     "@mycrypto/eth-scan": "^2.1.0",
-    "@types/redux-logger": "^3.0.7",
     "bs58": "4.0.1",
     "chalk": "2.4.2",
     "classnames": "^2.2.6",
     "compression": "1.7.4",
     "copy-webpack-plugin": "^5.0.0",
     "cross-env": "5.2.0",
     "cryptocompare": "^1.0.0",
-    "dotenv": "^6.1.0",
```

Comment:
Waaaaait, don't we use dotenv all over the place?

Reply:
Yes, but we get it via the required dependencies of dotenv-webpack -> dotenv-defaults

Reply:
Makes sense.